Lib/fontTools/otlLib/builder.py

```diff
@@ -668,6 +1204,20 @@ def buildLigatureSubstSubtable(mapping):
 
 
 def buildAnchor(x, y, point=None, deviceX=None, deviceY=None):
+    """Builds an Anchor table.
+
+    This determines the appropriate anchor format based on the passed parameters.
+
+    Args:
+        x (int): X coordinate.
+        y (int): Y coordinate.
+        point (int): Index of glyph contour point, if provided.
```

Comment:
I don't know if we want to use the typing annotation syntax in the docstrings or not (ideally yes, but perhaps can be done later?), e.g.

```suggestion
        point (Optional[int]): Index of glyph contour point, if provided.
```

If we do, we'll have to change this and many other similar places.